.ci/setup_env_alinux.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || "source /usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+# Obtain AlibabaCloud Linux version
+if [ -f /etc/os-release ]; then
```

Comment:
Either `/etc/os-release` or `/usr/lib/os-release` is source´ed on line 11 so that `VERSION_ID` is already exported on your env. So here you only need `alinux_version=$(echo $VERSION_ID | cut -d '"' -f2)`